cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java

```diff
@@ -38,6 +38,9 @@
     @ApiModelProperty( value = "Billable entries for the estimate", readOnly = true )
     private List<OrderLineDTO> lines = new LinkedList<>();
 
+    @ApiModelProperty( value = "Lines billed at the end of the billing period", readOnly = true )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "readOnly"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)